ci/README.md

```diff
@@ -13,71 +13,28 @@ binary built from the latest tip of master that passed tests.
 
 ## Alpine envoy image
 
-Minimal images based on alpine Linux allow for quicker deployment of Envoy. Two alpine based images are built,
+Minimal images based on Alpine Linux allow for quicker deployment of Envoy. Two Alpine based images are built,
 one with an Envoy binary with debug (`lyft/envoy-alpine-debug`) symbols and one stripped of them (`lyft/envoy-alpine`).
 Both images are pushed with two different tags: `<hash>` and `latest`. Parallel to the Ubuntu images above, `<hash>` corresponds to the
 master commit at which the binary was compiled, and `latest` corresponds to a binary built from the latest tip of master that passed tests.
 
-# Building a debug image
-An example basic invocation to build a debug image and run all tests is:
+# Building and running tests as a developer
 
-```bash
-docker pull lyft/envoy-build:latest && docker run -t -i -u $(id -u):$(id -g) -v <BUILD_DIR>:/build -v <SOURCE_DIR>:/source lyft/envoy-build:latest /bin/bash -c "cd /source && ci/do_ci.sh debug"
-```
-
-On OSX using the command below may work better. Unlike on Linux, users are not
-synced between the container and OS. Additionally, the bind mount will not
-create artifacts with the same ownership as in the container ([read more about
-osxfs][osxfs]).
+An example basic invocation to build the Envoy static binary (using the Bazel `fastbuild` type) is:
 
 ```bash
-docker pull lyft/envoy-build:latest && docker run -t -i -u root:root -v <BUILD_DIR>:/build -v <SOURCE_DIR>:/source lyft/envoy-build:latest /bin/bash -c "cd /source && ci/do_ci.sh debug"
+./ci/run_envoy_docker.sh bazel.fastbuild
 ```
 
-This bind mounts `<SOURCE_DIR>`, which allows for changes on the local
-filesystem to be consumed and outputs build artifacts in `<SOURCE_DIR>/build`.
-The static Envoy binary can be found in `<SOURCE_DIR>/build_debug/source/exe/envoy`.
+The output artifacts can be found in `/build/envoy/bazel-bin/source/exe/envoy-static` in the
+container.
 
 The `do_ci.sh` targets are:
```

Comment:
Should references to do_ci.sh in the docs be updated to point to run_envoy_docker.sh ?